README.md

```diff
@@ -1,15 +1,19 @@
 # fresh_charts
 
-A server side rendered charting library for Fresh based on
+A full stack charting library for Fresh based on
```

Comment:
'full stack' charting library sounds confusing to me. How about `A charting library for Fresh based on Chart.js, works in both server and browser` (or similar)?

Reply:
thanks for the feedback! updated the docs accordingly.